dot/parachain/candidate-validation/mocks_generate_test.go

```diff
@@ -4,4 +4,5 @@
 package candidatevalidation
 
 //go:generate mockgen -destination=mocks_test.go -package=$GOPACKAGE . PoVRequestor
-//go:generate mockgen -destination=mocks_runtime_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/parachain/runtime RuntimeInstance,ValidationHost
+//go:generate mockgen -destination=mocks_blockstate_test.go -package=$GOPACKAGE . BlockState
+//go:generate mockgen -destination=mocks_instance_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/lib/runtime Instance
```

Comment:
I had used a subset of methods from the original `Runtime` instance in `RuntimeInstance`.

Do we need all the methods?

Reply:
In the process from chainstate test I'm mocking block state to make a mock call to GetRuntime, and mock blockState expects the return to implement the full runtime interface.  When I try without I get this error 
```
wrong type of argument 0 to Return for *candidatevalidation.MockBlockState.GetRuntime: candidatevalidation.MockRuntimeInstance is not assignable to runtime.Instance
```